src/OpenTelemetry.Exporter.Prometheus/Implementation/PrometheusCollectionManager.cs

```diff
@@ -0,0 +1,215 @@
+// <copyright file="PrometheusCollectionManager.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry.Metrics;
+
+namespace OpenTelemetry.Exporter.Prometheus
+{
+    internal sealed class PrometheusCollectionManager
+    {
+        private readonly PrometheusExporter exporter;
+        private readonly int scrapeResponseCacheDurationInMilliseconds;
```

Comment:
minor: consider the following name to align with https://github.com/open-telemetry/opentelemetry-dotnet/blob/5e369a6af1e0d7fd0d03bdd5035e7fe325ae4d98/src/OpenTelemetry/BatchActivityExportProcessor.cs#L27.

```suggestion
        private readonly int scrapeResponseCacheDurationMilliseconds;
```

Reply:
Updated